pages/charcuterie.js

```diff
@@ -76,6 +77,13 @@ export default function Charcuterie() {
         </ContentContainer>
         <Footer />
       </main>
+      <Carousel images={['https://upload.wikimedia.org/wikipedia/commons/thumb/1/11/Test-Logo.svg/783px-Test-Logo.svg.png', 'https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg', 'https://www.rd.com/wp-content/uploads/2021/01/GettyImages-588935825.jpg']}
+                height={180}
+                width={260} />
+
+      <div style={{ position: 'fixed', top: 0, background: '#000000', width: '100%' }}>
```

Comment:
I'm not sure if we missed it on the Figma file, but were there design notes on what the behaviour of the nav bar should be? What we have done for previous sites is fade out the navbar as the user scrolls down and fade it back in on scroll up (check https://nwhacks.io for an example).

If not then we should ask design what they think! Comments on the Figma file or messaging in the design channel should be good :))

Reply:
For the record if we decide to have some show/hide behaviour this can go into a separate PR to prevent this from becoming too chonky, but do whatever works best for you!

Reply:
Nit: can you set `background` to `'rgb(0, 0, 0, 0.50)'`? Just so we can see what's behind the nav bar for the time being!

Reply:
changed the background! Nothing on figma but I can ask and am happy to work on the ticket:)

Reply:
messaged!

Reply:
gonna implement the fade effect

Reply:
Opened a new issue for the fade effect in #52, this comment is resolved :)) thanks!